rcl/include/rcl/error_handling.h

```diff
@@ -18,44 +18,32 @@
 #include "rcutils/error_handling.h"
 
 /// The error handling in RCL is just an alias to the error handling in rcutils.
-/**
- * Allocators given to functions in rcl are passed along to the error handling
- * on a "best effort" basis.
- * In some situations, like when NULL is passed for the allocator or something
- * else that contains it, the allocator is not available to be passed to the
- * RCL_SET_ERROR_MSG macro.
- * In these cases, the default allocator rcl_get_default_allocator() is used.
- * Since these are considered fatal errors, as opposed to errors that might
- * occur during normal runtime, is should be okay to use the default allocator.
- */
 
 typedef rcutils_error_state_t rcl_error_state_t;
+typedef rcutils_error_string_t rcl_error_string_t;
 
-#define rcl_error_state_copy rcutils_error_state_copy
-
-#define rcl_error_state_fini rcutils_error_state_fini
+#define rcl_initialize_error_handling_thread_local_storage \
```

Comment:
I just want to confirm that this function doesn't need to be called anywhere in `rcl` (assuming there's no threading happening). Should it be used in the other client libraries (e.g. rclcpp)?